docs/guide/algob-console.md

```diff
@@ -336,7 +336,7 @@ let validParams = {
   amount: 500,
   assetID: 'gold', // passing asa name is also supported
   lsig: lsigGoldOwner,
-  payFlags: { totalFee: 1000 }
+  payFlags: { totalFee: 1000, flatFee: true }
```

Comment:
shall we have that as a default? This way we won't need to set it all the time.

The code would be:
```
if (payFlags.flatFee === undefined) payFlags.flatFee = true
```

Reply:
OK

Reply:
```
  if(userParams.flatFee === undefined) {
    if(userParams.totalFee !== undefined) s.flatFee = true;
    else s.flatFee = false;
  }
```
Updated